park_api/cities/Heidelberg.py

```diff
@@ -0,0 +1,68 @@
+from bs4 import BeautifulSoup
+from park_api.util import convert_date
+from park_api.geodata import GeoData
+import datetime
+import json
+import urllib
+import random
+
+# This loads the geodata for this city if <city>.geojson exists in the same directory as this file.
+# No need to remove this if there's no geodata (yet), everything will still work.
+geodata = GeoData(__file__)
+
+# This function is called by the scraper and given the data of the page specified as source in geojson above.
+# It's supposed to return a dictionary containing everything the current spec expects. Tests will fail if it doesn't ;)
+def parse_html(html):
+
+    # BeautifulSoup is a great and easy way to parse the html and find the bits and pieces we're looking for.
+    # soup = BeautifulSoup(html, "html.parser")
+
+    # read the JSON-file:
+    #      URL + no caching
+    urlHD = "http://parken.heidelberg.de/api-v1/parking-location?api_key=H5WaIyR4lgn6wzo7rJf8u4ubecgpX0Q8&nc="+str(random.random())
+    headerHD={'Accept': 'application/json; charset=utf8'}
```

Comment:
Nur um im Falle der Fälle für den API-Betreiber einfacher ansprechbar zu sein, könntest du hier noch den User-Agent setzen? Siehe folgenden Code in der Datei scraper.py. Das konntest du natürlich nicht wissen, in Frankfurt fehlt das ebenso wie ich gerade sehe 🙈 

https://github.com/offenesdresden/ParkAPI/blob/1b103085b0db3973e41d2448cd939d64cbdbd6d1/park_api/scraper.py#L7-L12

Reply:
wo kommt das rein?

Im py-Skript setze ich bereits einen Header: ` headerHD={'Accept': 'application/json; charset=utf8'}`
Soll ich dieses Statement entsprechend erweitern?

Reply:
so etwa:
    `headerHD={'Accept': 'application/json; charset=utf-8', "User-Agent": "ParkAPI v%s - Info: %s" %(env.SERVER_VERSION, env.SOURCE_REPOSITORY) }`

Reply:
done.

Reply:
Super, danke 👍